### DISCUSSION TITLE
cant import simbrief from mcdu aoc

### DISCUSSION THREAD

**Discussion by monkeh88** [flybywiresim/aircraft#5921] [Help]
when i open the mcdu , aoc , i cant see simbrief there, when i try to import the simbrief flightplan in flypad its not importing either, how i fix this? https://www.youtube.com/watch?v=ExOAM-m-u8s

**Comment by GhostEagle68** [comment]
Simbrief is in EFB options now. Type in your ID (found on the FP before generated) and you're good to go!
